pkg/stream/producer.go

```diff
@@ -32,13 +33,13 @@ func NewProducer(config cfg.Config, logger log.Logger, name string, handlers ...
 	var output Output
 
 	if !settings.Daemon.Enabled {
-		if output, err = NewConfigurableOutput(config, logger, settings.Output); err != nil {
+		if output, err = NewConfigurableOutput(context.TODO(), config, logger, settings.Output); err != nil {
```

Comment:
So we want to have a proper context here eventually?

Reply:
This change would need a lot of changes in the upstream projects.. but yeah.. just let us go through with it..